samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -793,14 +794,14 @@ The device is identified using its UUID rather than its IMEI, since both overlay
 Building with nRF7002 EK Wi-Fi scanning support (for nRF91 Series DK)
 =====================================================================
 
-To build the sample with nRF7002 EK Wi-Fi scanning support, use the ``-DSHIELD=nrf7002ek`` and  ``-DEXTRA_CONF_FILE=overlay-nrf7002ek-wifi-scan-only.conf`` options.
+To build the sample with nRF7002 EK Wi-Fi scanning support, use the ``-DSHIELD=nrf7002ek``, ``-DSB_CONF_FILE=sysbuild_wifi_scan.conf`` and  ``-DEXTRA_CONF_FILE=sysbuild_nrf7002ek-wifi-scan.conf`` options.
```

Comment:
Does the `-DSB_CONF_FILE` need to be `sysbuild_nrf700x-wifi-scan.conf` here?

Reply:
Good catch, thank you